app/src/main/res/drawable/ic_shortcut_course.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="48dp"
+    android:height="48dp"
+    android:viewportWidth="48"
+    android:viewportHeight="48">
+  <path
+      android:pathData="M24,46L24,46C11.8,46 2,36.2 2,24v0C2,11.8 11.8,2 24,2h0c12.2,0 22,9.8 22,22v0C46,36.2 36.2,46 24,46z"
+      android:fillColor="#F5F5F5"/>
+  <path
+      android:pathData="M36,17.5c-0.2,0.3 -0.4,0.5 -0.6,0.6C32,21.1 28.5,24 25,26.9c-0.4,0.3 -1.2,0.4 -1.7,0.2c-2.6,-0.9 -5.2,-1.9 -7.8,-2.9c-0.5,-0.2 -0.9,-0.3 -1.7,-0.6c0,0.9 -0.1,1.7 0,2.4c0,0.3 0.4,0.7 0.7,0.9c2.9,1.3 5.8,2.5 8.7,3.8c0.7,0.3 1.2,0 1.7,-0.4c2.8,-2.5 5.7,-5 8.5,-7.5c0.8,-0.7 1.5,-0.7 2.4,0c-0.3,0.4 -0.6,0.6 -0.8,0.9c-3.2,3 -6.5,5.9 -9.7,8.9c-0.6,0.6 -1.1,0.7 -1.9,0.3c-3.5,-1.6 -6.9,-3.1 -10.4,-4.6c-0.8,-0.3 -1.2,-0.8 -1.1,-1.7c0.1,-1.3 -0.1,-2.6 0,-3.8c0,-0.6 0.4,-1.3 0.9,-1.6c3.5,-2.3 7,-4.5 10.5,-6.7c0.4,-0.3 1.1,-0.4 1.5,-0.3c3.6,1 7.1,2.1 10.7,3.2C35.7,17.2 35.8,17.4 36,17.5z"
```

Comment:
You use the icon for _all courses_ here. Let's use the icon for _my courses_ instead. :)